pytorch_pfn_extras/onnx/annotate.py

```diff
@@ -191,7 +193,9 @@ def annotate(**attrs):
     Args:
         attrs (dict): annotation parameters
     """
-    return _Annotation(**attrs)
+    if torch.onnx.is_in_onnx_export():
```

Comment:
So `annotate`  and `scoped_anchor` would be ignored when it's not exporting onnx
👍